website/static/js/licenses.js

```diff
@@ -0,0 +1,44 @@
+var $ = require('jquery');
+
+var licenses = $.map(require('list-of-licenses'), function(value, key) {
+    value.id = key;
+    var properties = value.properties || [];
+    var newProperties = {};
+    $.each(properties, function(i, prop) {
+        var words = prop.split(' ');
+        words[0] = words[0].slice(0, 1).toLowerCase() + words[0].slice(1);        
```

Comment:
Might be better to just add these "IDs" in the list-of-licenses repo? We shouldn't really have to do much preprocessing of the data; you are the owner of the repo, after all.